runtime/src/bridge.rs

```diff
@@ -13,7 +14,10 @@ type ResourceId = chainbridge::ResourceId;
 type BalanceOf<T> =
     <<T as Trait>::Currency as Currency<<T as frame_system::Trait>::AccountId>>::Balance;
 
-pub trait Trait: system::Trait + chainbridge::Trait {
+/// Additional Fee charged when moving native tokens to target chains (RAD)
+const TOKEN_FEE : u32 = 20;
```

Comment:
Wondering if I should make this a storage value, to modify it faster if needed.

Reply:
I think in this case is an overkill to use the fee storage module. Since I will need to store as well the key separately, that would be two items to store. The key to use and the fee. 
I would rather just store the RAD amount.

Reply:
Either is fine for me, but I guess you are right – we can always make fees adjustable in an on-chain upgrade. If we don't plan to change them anytime soon and if there's no economic risk of being attacked (i. e. fees are way too low), I'd suggest to keep the fees static for now. Any thoughts @lucasvo ?

Reply:
IMO, It would not be a lot storage since we will not have many keys. This can also be a single place for all configurable ones instead of having new API for each value. Either way, your call

Reply:
The main reason for me to add just this fees storage is in case that the amount is too big/small that we need to react fast. My idea was to only add storage for the fee of 10 and 20 RAD ones for nft and native transfer, so we could do a quick storage update if needed. 
But maybe we are overengineering it, and in a few months once the bridge cost goes down, we can remove this fees altogether